libraries/crypto/Cargo.toml

```diff
@@ -11,7 +11,6 @@ edition = "2018"
 
 [dependencies]
 libtock_drivers = { path = "../../third_party/libtock-drivers" }
-cbor = { path = "../cbor" }
```

Comment:
I guess this is because the PR starts from an older commit.

Reply:
It was more that I couldn't see any use of `cbor` in this directory (and that doesn't seem to have changed recently) &ndash; am I missing something?

Reply:
Probably removed. I have something in the back of my mind. Go ahead removing it.